site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -160,7 +168,22 @@ export const CreateWorkspacePageView: FC<React.PropsWithChildren<CreateWorkspace
                 </Stack>
               )}
 
-              <FormFooter onCancel={props.onCancel} isLoading={props.creatingWorkspace} />
+              {props.quota ? (
+                <WorkspaceQuota
+                  quota={props.quota}
+                  error={
+                    props.createWorkspaceErrors[CreateWorkspaceErrors.GET_WORKSPACE_QUOTA_ERROR]
+                  }
+                />
+              ) : (
+                <></>
```

Comment:
Same comment as above

```suggestion
                null
```